package.json

```diff
@@ -2,18 +2,16 @@
   "name": "react-live",
   "version": "1.9.2",
   "description": "A production-focused playground for live editing React code",
-  "main": "lib/index.js",
+  "main": "dist/react-live.cjs.js",
```

Comment:
Is there any use case where a CJS consumer wants less than all of React Live? Because you can't tree shake CJS in most cases...

Reply:
@ryan-roemer Possibly; It's all up to uglify & webpack. The reason why I think it makes sense here is that in a typical environment you'd import all of the components

Reply:
SGTM!